docs/operator-guide/server-setup.md

```diff
@@ -28,10 +38,11 @@ the point where the new empty CN first boots.
 
 ## First Boot
 
-When a CN boots (whether by PXE or USB) the first interaction it will have with
-the Triton control plane will be via DHCP. It will do a DHCPDISCOVER in order to
-attempt to learn its own IP address. This DHCPDISCOVER is a broadcast message
-and the booter/dhcpd service (usually dhcpd0 zone on the HN) will see this
+When a CN boots (whether by PXE, USB, or from disk) the first
+interaction it will have with the Triton control plane will be via
+DHCP. It will do a DHCPDISCOVER in order to attempt to learn its own
+IP address. This DHCPDISCOVER is a broadcast message and the
+booter/dhcpd service (usually dhcpd0 zone on the HN) will see this
 message be responsible for handling it.
```

Comment:
Not your typo but 'see this message be responsible' --- I think 'and be responsible' is probably what was meant (if you want to fix).

Reply:
Will fix.